### ISSUE TITLE
File accuracy after model training

### ISSUE THREAD

**Issue by ydi-mua** [graphdeco-inria/gaussian-splatting#512]
hi，I downloaded the model.zip file you provided and found that the trained ply files in it are very large, such as the garden model (more than 1 GB), and the results are very good. But my own training files are not very big. There are nearly 1,000 training photos. Are there any parameters or other settings?

**Comment by hariharan1412** [comment]
Yeah, I'm also getting similar output as yours. It'll be helpful if someone tells me why this is happening.
